### ISSUE TITLE
Add pre-trained agent models

### ISSUE THREAD

**Issue by generic-github-user** [QuantumInformation/youtube-space-invaders#12]

**Comment by quantuminformation** [comment]
Shall I set up some training for this on some cloud instance?

**Comment by generic-github-user** [comment]
Sure, but there's some things that need to be finished up before training can be done - mostly this: https://github.com/QuantumInformation/youtube-space-invaders/issues/18

**Comment by quantuminformation** [comment]
Then again, how long do you think a reasonably priced laptop with a dedicated graphics card could produce results. Or would even integrated graphics do the job.

I ran the game a few times and each game takes about 1-3 minutes at normal speed. However, if we increase the game speed do you think the accuracy will suffer for the model that would be used to play the normal speed?

**Comment by generic-github-user** [comment]
It's tough to say without testing . . . the results would probably be in the same order of magnitude as something tested on OpenAI gym, but it's going to be slower since it's web-based. If I remember correctly, I added a speed control to the game so that we can train the bot without it being constrained by the game running slowly (at the speed a human would play it at).

**Comment by quantuminformation** [comment]
Yeah I remember, I'll run it for a few hours at a faster speed and get back to you.

**Comment by generic-github-user** [comment]
Well the training isn't finished yet so I don't know if it'll do much . . . I think the critic training might be finished but I don't remember how well it worked.
